admin_guide/limits.adoc

```diff
@@ -87,7 +87,36 @@ specified.
 <13> The default amount of memory that a container will request to use if not specified.
 <14> The maximum amount of CPU burst that a container can make as a ratio of its limit over request.
 ====
-// end::admin_limits_sample_definition[]
+
+.OpenShift Limit Range Object Definition
+====
+[source,yaml]
+----
+apiVersion: "v1"
+kind: "LimitRange"
+metadata:
+  name: "openshift-resource-limits"
+spec:
+  limits:
+    -
```

Comment:
By **object** in this case, do you mean associative array (aka dictionary, aka hash table, aka ...)?  Are you sure that it **cannot** be on the same line?  I see many instances of that (docs, code, etc) and so far they seem to work fine...